tests/examples/source/devfiles/nodejs/devfile-for-run.yaml

```diff
@@ -0,0 +1,75 @@
+schemaVersion: 2.2.0
+metadata:
+  name: nodejs
+  projectType: nodejs
+  language: nodejs
+starterProjects:
+  - name: nodejs-starter
+    git:
+      remotes:
+        origin: "https://github.com/odo-devfiles/nodejs-ex.git"
+components:
+  - name: runtime
+    container:
+      image: registry.access.redhat.com/ubi8/nodejs-12:1-36
+      memoryLimit: 1024Mi
+      endpoints:
+        - name: "3000-tcp"
+          targetPort: 3000
+      mountSources: true
```

Comment:
Can we also add a case with one more side container component in which we want to run commands? I think this can be interesting to cover..